flask_app/resources/nearby_restaurants.py

```diff
@@ -0,0 +1,21 @@
+from flask_restful import Resource, abort
+from models import db, Address, Restaurants
+from sqlalchemy import func
+
+
+class NearbyRestaurantsResource(Resource):
+    def get(self, user_id):
+        radius = 5000
```

Comment:
Should radius be hardcoded? Wasn't our initial idea to create some kind of delivery_tiers or sth like that. If company choses more expensive contract it will have bigger delivery radiu and some other stuff.

Reply:
I see, so we draw 5km radius for user. Hmm, do you think it's alright? I think that even though it might be a bit more complicated we should stick to the idea that the company specifies their delivery radius. In the future I think it will us more good. 

I understand it's not the first thing to think about now. But we should create a issue with restaurant specifying their radius instead of hard coded user-centered radius.

Reply:
I will create this issue, fill free to take it in your spare time.

Reply:
Yeah, this version is most basic one to already provide code for our frontend developer. I will update this with radius being declared by restaurant itself, you can expect those changes in few more days. Let me know what you think about this idea

Reply:
I like that, but first please focus on more pressing issues